src/zipkin.js

```diff
@@ -30,9 +32,24 @@ export function truncate(zspan) {
 export class PatchedZipkinExporter {
   constructor(beaconUrl) {
     this.beaconUrl = beaconUrl;
+    this.spanCounts = {};
+    const exporter = this;
+    setInterval(() => {
+      exporter.spanCounts = {};
+    }, SPAN_RATE_LIMIT_PERIOD);
+  }
+
+  filter(span) {
+    const component = span.attributes ? span.attributes.component || 'unknown' : 'unknown';
```

Comment:
How about `(span.attributes && span.attributes.component) || 'unknown'`?

Reply:
Lint didn't seem to like that.  Tried an alternate that doesn't repeat the constant at least.